### DISCUSSION TITLE
适配苹果隐私新规，可以提供一份隐私清单吗？

### DISCUSSION THREAD

**Discussion by z-shu07** [Tencent/libpag#2255] [Q&A]
请提供一份隐私清单，用于 APP 提审，不胜感谢

**Comment by z-shu07** [comment]
@kevingpqi123

**Reply by kevingpqi123** [reply]
https://pag.art/docs/sdk.html

<img width="532" alt="image" src="https://github.com/Tencent/libpag/assets/16609221/f496a89c-7db0-482b-9f4c-05a33518e1c9">

**Reply by z-shu07** [reply]
你好，需要的是一个 PrivacyInfo.xcprivacy文件，对隐私API使用范围进行说明，而不是上面的个人信息保护规则。
示例：https://cloud.tencent.com/document/product/269/104138?from_column=20421&from=20421

**Reply by kevingpqi123** [reply]
暂时还没有，我们下周处理下这部分内容
